System/Posix/Files.hsc

```diff
@@ -290,7 +290,7 @@ rename :: FilePath -> FilePath -> IO ()
 rename name1 name2 =
   withFilePath name1 $ \s1 ->
   withFilePath name2 $ \s2 ->
-  throwErrnoPathIfMinus1_ "rename" name1 (c_rename s1 s2)
+  throwErrnoIfMinus1_ ("rename "++name1++" to "++name2) (c_rename s1 s2)
```

Comment:
Filenames can include spaces, I usually find such messages more readable and unambiguous when paths are enclosed in single quotes. 

How about a shared helper function for `rename` / `createLink` / `createSymbolicLink`?
```haskell
throwErrnoTwoPathsIfMinus1_ loc path1 path2 = 
  throwErrnoIfMinus1_ (loc ++ "'" ++ path1 ++ "' to '" ++ path2 ++ "'")
```

Reply:
I have no strong opinion on these issues. I'm happy to just do it as you prefer.